tests/connectors/luoo.js

```diff
@@ -1,7 +1,21 @@
 'use strict';
 
 module.exports = function(driver, connectorSpec) {
+	// /vol or /music
 	connectorSpec.shouldBehaveLikeMusicSite(driver, {
 		url: 'http://www.luoo.net/music/854'
 	});
+	// essay
+	connectorSpec.shouldBehaveLikeMusicSite(driver, {
+		url: 'http://www.luoo.net/essay/915'
+	});
+	// musician
+	connectorSpec.shouldBehaveLikeMusicSite(driver, {
+		url: 'http://www.luoo.net/musician/'
+	});
+	// search
+	connectorSpec.shouldBehaveLikeMusicSite(driver, {
+		url: 'http://www.luoo.net/search/?q=love&t=single',
+		playButtonSelector: '.icon-search-play:eq(-1)'
```

Comment:
WebDriver throws an error when clicks on an element with such selector. Do you have the same issue?

I tried to use `.icon-search-play` selector and it works fine for me, but there's another thing that first three songs from search results are unplayable for me. Clicking on play icon does nothing. I updated the test like this way and it passed for me:
```js
	// search
	connectorSpec.shouldBehaveLikeMusicSite(driver, {
		url: 'http://www.luoo.net/search/?q=Spinning&t=single',
		playButtonSelector: '.icon-search-play'
	});
```